spec/models/user_spec.rb

```diff
@@ -16,6 +16,19 @@
   it { should have_valid(:state).when('MA', 'VT')}
   it { should_not have_valid(:state).when(nil, '')}
 
+  it { should have_valid(:role).when('member', 'admin')}
+  it { should_not have_valid(:role).when(nil, '')}
+  
+  it "is not an admin if the role is not admin" do
+    user = FactoryGirl.create(:user, role: "member")
+    expect(user.admin?).to eq(false)
+  end
+
+  it "is an admin if the role is admin" do
+    user = FactoryGirl.create(:user, role: "admin")
```

Comment:
You could also use `FactoryGirl` `traits` to register a separate factory for admins.